### ISSUE TITLE
Download options

### ISSUE THREAD

**Issue by JaxonPlays** [preservetube/preservetube#7]
Download .description
Download .info.json
Download .webp
Download .webm

<sub>[PRSV-12](https://linear.app/tndllc/issue/PRSV-12/download-options)</sub>

**Comment by unknownsrc** [comment]
Good idea. I'd have to see how I could implement this on the frontend, as having a bunch of download buttons next to each other would be super annoying.

**Comment by JaxonPlays** [comment]
That’s exactly what I want.

Example:![image](https://github.com/preservetube/preservetube/assets/47770776/9c23b419-f42b-409d-ada4-9e372a7e17bd)
